src/Framework/PaymentGateways/Log/PaymentGatewayLog.php

```diff
@@ -24,6 +25,21 @@ public static function __callStatic($name, $arguments)
         foreach ($arguments[1] as $argument) {
             if ($argument instanceof GatewayPaymentData) {
                 unset($argument->cardInfo, $argument->legacyPaymentData['card_info']);
+
+                $cardFields = [
+                    'card_number',
+                    'card_cvc',
+                    'card_name',
+                    'card_exp_month',
+                    'card_exp_year',
+                    'card_expiry'
+                ];
+
+                foreach ($argument->legacyPaymentData['post_data'] as $fieldName => $data) {
+                    if (in_array($fieldName, $cardFields, true)) {
+                        unset($argument->legacyPaymentData['post_data'][$fieldName]);
+                    }
+                }
```

Comment:
@ravinderk how valuable do you think logging the `legacyPaymentData` property is?  Isn't it just a duplicate of the entire request just shaped specifically for legacy items?

If that's the case then we technically already have the data we need inside the DTO so maybe we we can just simplify this all together with:

`unset($argument->legacyPaymentData);`